.changeset/wicked-rings-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/prosemirror-editor": minor
+---
+
+Add option to disable editors
```

Comment:
Just to clarify I understand the release process, it looks from the [documentation](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) that this file gets generated automatically by running `yarn changeset`. Then the [changesets GitHub Action](https://github.com/changesets/action) will automatically open a release PR which I assume will convert the changeset info into a changelog? (example [here](https://github.com/guardian/cdk/pull/2105))

Reply:
Yes - I was prompted to run `yarn changeset` after my first commit, `changesets` said there wasn't a 'changeset' and this added one, which I added in a follow-up commit.